cnxdb/tests/cli/test_main.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+
+import psycopg2
+import pytest
+
+from .. import testing
+
+
+def _translate_parts_to_args(parts):
+    """Translates connection string parts to arguments"""
+    return ['-h', parts['host'],
+            '-p', parts['port'],
+            '-d', parts['dbname'],
+            '-U', parts['user'],
+            ]
+
+
+def test_init(connection_string_parts, db_wipe):
```

Comment:
I had the above idea (using `db` instead of `db_init` and `db_wipe`) from https://pytest.org/latest/fixture.html, specifically I looked at these particular examples:

```python
import smtplib
import pytest

@pytest.fixture(scope="module")
def smtp(request):
    smtp = smtplib.SMTP("smtp.gmail.com")
    def fin():
        print ("teardown smtp")
        smtp.close()
    request.addfinalizer(fin)
    return smtp  # provide the fixture value
```

```python
import pytest

class DB:
    def __init__(self):
        self.intransaction = []
    def begin(self, name):
        self.intransaction.append(name)
    def rollback(self):
        self.intransaction.pop()

@pytest.fixture(scope="module")
def db():
    return DB()
```